app/app.go

```diff
@@ -897,57 +921,6 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	return paramsKeeper
 }
 
-func (app *App) InitializeAppVersion(ctx sdk.Context) {
-	appVersion := app.GetAppVersionFromParamStore(ctx)
-	if appVersion == 0 {
-		// if the param store does not have an app version set, default to v1
-		app.SetAppVersion(ctx, v1)
-	} else {
-		app.SetAppVersion(ctx, appVersion)
-	}
-}
-
-// OfferSnapshot is a wrapper around the baseapp's OfferSnapshot method. It is
```

Comment:
This seems redundant in 0.52 since there's no way to obtain the AppVersion from an `abci.RequestOfferSnapshot`.  Not totally clear why it is needed to lazy mount stores, or offer snapshots for an arbitrary consensus version (maybe for syncing?) but if it is we'll need a workaround.   Maybe a custom request type is needed?